package/upgrade/upgrade_node.sh

```diff
@@ -362,6 +355,53 @@ clean_rke2_archives() {
     done
 }
 
+convert_nodenetwork_to_vlanconfig() {
+  [[ $UPGRADE_PREVIOUS_VERSION != "v1.0.3" ]] && return
```

Comment:
`UPGRADE_PREVIOUS_VERSION` is only available after calling `detect_upgrade()`

Reply:
This could be the reason why there's no `vlanconfig` created after the upgrade as I mentioned [here](https://github.com/harvester/harvester/pull/2852#issuecomment-1275922610).